docs/protocol/specifications/data.md

```diff
@@ -380,17 +390,17 @@ Serialization flags control what and how data is encoded in a given *Transaction
 
 The **first (least significant) bit** indicates whether the transaction includes witness data. If set to zero, the input and output witness fields are absent.
 
-The **second bit** indicates whether the output commitment from the spent output is present in the [input spend commitment](#asset-version-1-spend-commitment). If set to zero, the output commitment field is absent.
+The **second bit** indicates whether the output commitment from the spent output is present in the [input spend commitment](#asset-version-1-spend-commitment). If set to zero, the output commitment field is replaced by the [SHA3-256](#sha3) hash of the commitment.
```

Comment:
As we've discussed elsewhere, it would be good to be super clear (here and throughout the spec) about when things like "the commitment" and "the witness" do and do not include the extensible-string length prefix.